app/javascript/component/App.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Link, Route, Routes } from 'react-router-dom';
+import Greeting from './greeting';
+
+const App = () => (
+    <div>
+        <h2 className='h2'>Refresh page to display random message</h2>
+        <Routes>
+            <Route path="/" element={<Greeting />} />
+        </Routes>
+    </div>
+)
+
+export default App;
```

Comment:
- [ ] Kindly make sure that your app is running properly. Currently, when I run the app, I get the following error which prevents me from trying your app.
![image](https://user-images.githubusercontent.com/62290545/209418849-20f91708-1931-44fb-8803-01a27a22a1a4.png)

- [HINT] This error is caused by `cors`, you can read more about it, how it works, and how to fix it from [here](https://www.stackhawk.com/blog/rails-cors-guide/)

Reply:
Solved